internal/observability/middleware_test.go

```diff
@@ -60,6 +61,7 @@ func TestMiddlewareLogging(t *testing.T) {
 		return []zapcore.Field{
 			zap.String("source", req.Caller),
 			zap.String("dest", req.Service),
+			zap.String("transport", unknownIfEmpty(req.Transport)),
```

Comment:
For consistency, all assertions should hard-code a string or all should use `unknownIfEmpty`. We're doing one here and a different thing below.

Reply:
Never mind, I'm wrong.